R/sampling.R

```diff
@@ -146,3 +144,233 @@ sample_combinations <- function(ntrain, ntest, nsamples, joint_sampling = TRUE)
 
   return(ret)
 }
+
+#' Sample ctree variables from a given conditional inference tree
+#'
+#' @param tree List. Contains tree which is an object of type ctree built from the party package.
+#' Also contains given_ind, the features to condition upon.
+#'
+#' @param n_samples Numeric. Indicates how many samples to use for MCMC.
+#'
+#' @param x_test Matrix, data.frame or data.table with the features of the observation whose
+#' predictions ought to be explained (test data). Dimension \code{1xp} or \code{px1}.
+#'
+#' @param x_train Matrix, data.frame or data.table with training data.
+#'
+#' @param p Positive integer. The number of features.
+#'
+#' @param sample Boolean. True indicates that the method samples from the terminal node
+#' of the tree whereas False indicates that the method takes all the samples if it is less than n_samples.
```

Comment:
I would write: " whereas False indicates that the function takes all the observations if the number of observations is less than n_samples.

Reply:
Agree